test/controllers/AddAdditionalMetadataControllerSpec.scala

```diff
@@ -296,7 +296,7 @@ class AddAdditionalMetadataControllerSpec extends FrontEndTestHelper {
       )
       addAdditionalMetadataPageAsString should include("""    <p class="govuk-error-message" id="error-FoiExemptionCode">
           |        <span class="govuk-visually-hidden">Error:</span>
-          |        Search for and select at least one FOI exemption code(s)
+          |        Select at least one FOI exemption code(s)
```

Comment:
Is the `...(s)` needed on the word `code`. Make sense without it

Reply:
I think this was based on an old copy change. But makes sense to remove it.